vignettes/GenomicFiles.Rmd

```diff
@@ -0,0 +1,727 @@
+---
+title: "Introduction to GenomicFiles"
+authors:
+  - name: "Valerie Obenchain"
+  - name: "Michael Love"
+  - name: "Martin Morgan"
+  - name: "Sonali Kumari"
+    affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
+date: "Last modified: October 2014; Compiled: `r format(Sys.time(), '%B %d , %Y')`"
+vignette: >
+  %\VignetteIndexEntry{Introduction to GenomicFiles}
+  %\VignetteDepends{GenomicAlignments, RNAseqData.HNRNPC.bam.chr14}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+output:
+  BiocStyle::html_document
+---
+
+# Introduction
+  
+This vignette illustrates how to use the `r Biocpkg("GenomicFiles")` package for
+distributed computing across files. The functions in `GenomicFiles` manipulate
+and combine data subsets via two user-supplied functions, MAP and REDUCE. These
+are similar in spirit to `Map` and `Reduce` in base R. Together they provide a
+flexible interface to extract, manipulate, and combine data. Both functions are
+executed in the distributed step, which means results are combined on a single
+worker, not across workers.
+
+We assume the reader has some previous experience with R and with basic
+manipulation of ranges objects such as `GRanges` and `GAlignments` and file
+classes such as `BamFile` and `BigWigFile`. See the vignettes and documentation
+in `r Biocpkg("GenomicRanges")`, `r Biocpkg("GenomicAlignments")`, `r Biocpkg
+("GenomicAlignments")`, `r Biocpkg("Rsamtools")` and `r Biocpkg("rtracklayer")`
```

Comment:
I believe GenomicAlignments is duplicated here. Remove the second one that is broken up at the end of line 32 and at the start of line 33.